docs/docs/creating-global-styles.md

```diff
@@ -25,6 +25,9 @@ a {
 
 Then, include the stylesheet in your site's `gatsby-browser.js` file.
 
+> _Note: This solution works when including css as those styles are extracted when building the js but not for css-in-js.
```

Comment:
You're starting the `_` for emphasis, but you're not ending it. And therefore, it's not being rendered properly (as seen in the screenshot below). Maybe, you forgot to end it with `_`?
![image](https://user-images.githubusercontent.com/19631364/44734728-195e4100-aada-11e8-872f-72b598e29ef6.png)